content/en/blog/_posts/2025-04-23-Kubernetes-v1-33-Release/index.md

```diff
@@ -0,0 +1,473 @@
+---
+layout: blog
+title: 'Kubernetes v1.33: TBC'
+date: 2025-04-23T10:30:00-08:00
```

Comment:
Not sure if this value format works because we used to use `date` rather than `datetime` format.

Reply:
Thanks! I think this could do a bit of clarifying in the guideline doc, but my understanding is that the included timestamp helps with the build setup, and this was how it was done with the mid cycle blog as well
https://github.com/rytswd/k8s-website/blame/20874d914430fbdfa4152967aa80813a427a04a2/content/en/blog/_posts/2025-03-26-kubernetes-1.33-sneak-peek.md#L4